cmd/scollector/collectors/redis_unix.go

```diff
@@ -239,10 +239,53 @@ func c_redis() (opentsdb.MultiDataPoint, error) {
 			Add(&md, "redis."+sp[0], sp[1], tags, metadata.Unknown, metadata.None, "")
 		}
 		Add(&md, "redis.key_count", keys, tags, metadata.Gauge, metadata.Key, descRedisKeyCount)
+
+		info_command, info_command_err := c.Do("info", "commandstats")
+		if info_command_err != nil {
+			Error = info_command_err
+			continue
+		}
+		infoCmdSplit := strings.Split(string(info_command.([]uint8)), "\n")
+		for _, line := range infoCmdSplit {
+			line = strings.TrimSpace(line)
+			sp := strings.Split(line, ":")
+			if len(sp) < 2 {
+				continue
+			}
+			if strings.HasPrefix(sp[0], "cmdstat_") {
```

Comment:
Invert this logic: https://github.com/golang/go/wiki/CodeReviewComments#indent-error-flow